### ISSUE TITLE
Task 2134 (unexpected task requirements for me)

### ISSUE THREAD

**Issue by ssfxate** [AgileNix/coderslang-feedback#79]
### Requirements
```
Add 'Times New Roman' font to all elements in the index.html file.
The font in the <h1> heading should be italic.
```

But the lecture https://js.coderslang.com/lecture/213 doesn't have a part for how to apply a font to the whole document.

I'd found a couple of solutions on the Internet.
Solved it by
``` css
* {
  font-family: 'Times New Roman';
}
```
But tried to use this solution as well, but unsuccessfully.
Locally works well.
``` CSS
body {
  font-family: 'Times New Roman';
}
```

**Comment by amatiyko** [comment]
@ssfxate In the task, you should add a font-family property to all elements. 
There are three elements in the document: `<h1>`, `<p>`, `<a>`.  You can add font-family to each of these tags separately.
Other selectors, you have provided in your feedback will be covered later.

**Comment by ssfxate** [comment]
Hmm... Is it better to rephrase it to "each element"?

**Comment by amatiyko** [comment]
@ssfxate done. thank you.
Please, delete task2134 folder and then download tasks again to see changes.
